_episodes/09-documentation.md

```diff
@@ -483,19 +543,29 @@ napoleon_use_ivar = True
 {: .language-python}
 
 Once compiled, this will generate documentation for all files in the `molecool` source directory, as well as exclude
-generating documentation for the tests and versioneering files in the project. The `index.rst` file will be updated
-to include an `API Reference` page, and the resultant `.html` files can be viewed in the `_build` directory upon 
+generating documentation for the tests and versioneering files in the project. 
+The `index.rst` file will be updated to include an `API Reference` page, and the resultant `.html` files can be viewed in the `_build` directory upon 
 compilation.
 
-
 ## Hosting your documentation
 
 ### Read The Docs
-We recommend hosing your  documentation on [Read The Docs](https://readthedocs.org/). With this service, you can enable the building of your documentation every time you push to your repository.
-
-Go to the [Read the Docs](https://readthedocs.org/) website. Log in with your GitHub username and hook the repository to read the docs. Push to the repository, or trigger a build on the site. Because of the recent switch from `master` to `main`, you may get an error that your build has failed. In order to fix this, click "Admin" in the menu, then on the side bar click "Advanced Settings". Under "Default branch" choose "main". Trigger a build again after making this change.
-
-Unfortunately, your documentation build will fail again. This is because we need our dependencies installed on RTD. There is another file the CookieCutter has added which we must now modify. Add your dependencies (numpy and matplotlib) to `docs/requirements.yaml`. You should also add `sphinx-autoapi` under `pip only installs`. Your `requirements.yaml` will look like this:
+We recommend hosing your  documentation on [Read The Docs](https://readthedocs.org/). 
+With this service, you can enable the building of your documentation every time you push to your repository.
+
+Go to the [Read the Docs](https://readthedocs.org/) website. 
+Log in with your GitHub username and hook the repository to read the docs. 
+Push to the repository, or trigger a build on the site. Because of the recent switch from `master` to `main`, you may get an error that your build has failed. 
```

Comment:
```suggestion
Push to the repository, or trigger a build on the site.
Because of the recent switch from `master` to `main`, you may get an error that your build has failed. 
```